src/util/unwind.hpp

```diff
@@ -14,28 +9,31 @@ namespace mc {
 namespace util {
 
 #ifdef WITH_UNWIND
```

Comment:
done. I was thinking about this as I was implementing this. I have assumed that it is a `std::uint64_t`, which is how it is defined in libunwind, with a `static_assert` in `unwind.cpp`, that will break the build if that assumption is every broken.